### DISCUSSION TITLE
How do I update calendar events in node.js?

### DISCUSSION THREAD

**Discussion by Julianbullmagic** [googleapis/google-api-nodejs-client#3617] [Q&A]
const updateRes = await calendar.events.update({
              calendarId: 'primary',
              eventId: event.id,
              resource: event,
            });

This doesn't work. 

Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

1) Is this a client library issue or a product issue?
This is the client library for . We will only be able to assist with issues that pertain to the behaviors of this library. If the issue you're experiencing is due to the behavior of the product itself, please visit the [ Support page]() to reach the most relevant engineers.

2) Did someone already solve this?
  - Search the issues already opened: https://github.com/googleapis/google-api-nodejs-client/issues
  - Search the issues on our "catch-all" repository: https://github.com/googleapis/google-cloud-node
  - Search or ask on StackOverflow (engineers monitor these tags): http://stackoverflow.com/questions/tagged/google-cloud-platform+node.js

3) Do you have a support contract?
Please create an issue in the [support console](https://cloud.google.com/support/) to ensure a timely response.

If the support paths suggested above still do not result in a resolution, please provide the following details.

#### Environment details

  - OS:
  - Node.js version:
  - npm version:
  - `googleapis` version:

#### Steps to reproduce

  1. ?
  2. ?

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by juancruzfl** [comment]
A couple of these things may not allow your calendar to be updated properly : 
- [ ] Proper authorization provided
- [ ] Valid eventId
- [ ] Valid scopes (if your scope is set to read only, you wont be able to update the calendar)
- [ ] Valid Resource
